packages/palette/src/elements/MultiSelect/MultiSelect.tsx

```diff
@@ -165,12 +156,16 @@ export const MultiSelect: React.FC<MultiSelectProps> = ({
         </Options>
       )}
 
+      {required && !(error && typeof error === "string") && (
+        <RequiredField mt={0.5} ml={1} />
+      )}
+
       {error && typeof error === "string" && (
-        <Text variant="xs" mt={0.5} color="red100">
+        <Text variant="xs" mt={0.5} ml={1} color="red100">
```

Comment:
Are all errors indented like this now?

Reply:
yep... its a little nitpicky but matches how they look on the other components.